src/components/Splash.js

```diff
@@ -144,8 +159,33 @@ export default ({
   backgroundColor = theme.primary,
   page = `TORONTO`,
 }: Props) => {
+
+  useEffect(() => {
+    const guildCardRoot = document.getElementById(`guild-card-root`)
+    const guildCardUpcoming = document.getElementById(`guild-card-upcoming`)
+    const guildCardPast = document.getElementById(`guild-card-past`)
+
+    window.$guild.renderGuildCard(
+      guildCardRoot,
+      `torontojs`
+    )
+
+    window.$guild.renderGuildEventList(
+      guildCardUpcoming,
+      `torontojs`,
+      `UPCOMING`
+    )
+
+    window.$guild.renderGuildEventList(
+      guildCardPast,
+      `torontojs`,
+      `PAST`
+    )
```

Comment:
All of these have return a function to run when unmounted! (This is undocumented so no way you would've known this 😅)

So, you can do this to be safe (but technically doesn't matter as this `useEffect` is only mounting once):

```suggestion
    const unmountGuildCard = window.$guild.renderGuildCard(
      guildCardRoot,
      `torontojs`
    )

    const unmountUpcomingEventList = window.$guild.renderGuildEventList(
      guildCardUpcoming,
      `torontojs`,
      `UPCOMING`
    )

    const unmountPastEventList = window.$guild.renderGuildEventList(
      guildCardPast,
      `torontojs`,
      `PAST`
    )
    
    return () => {
      unmountGuildCard()
      unmountUpcomingEventList()
      unmountPastEventList()
    }
```